### PR TITLE
API break: allow multiple fallback address

### PR DESCRIPTION
**Pull request by rustyrussell**
Fixes: #590 

This breaks JSON API, so we'd better do it now.  We use the deprecated_apis trick to make things work for now.

### CODE REVIEW

File: doc/lightning-invoice.7.txt [hunk 1]
```diff
@@ -32,6 +32,10 @@ UTF-8, and cannot use '\u' JSON escape codes.
 The 'expiry' is optionally the number of seconds the invoice is valid for.
 If no value is provided the default of 3600 (1 Hour) is used.
 
+The 'fallbacks' array is one or more fallback addresses to include in
```
Comment (ZmnSCPxj) [inline doc/lightning-invoice.7.txt]: This paragraph is not up-to-date in the manpage file.

**Comment by cdecker** [comment]
Concept ACK, pending manpage fix

**Comment by rustyrussell** [comment]
Thanks, updated!

**Comment by cdecker** [comment]
CI failure was unrelated, will look at it later :wink: 
ACK 8bc9488

**Pull request merged**